src/objecttypes/core/admin.py

```diff
@@ -139,3 +151,25 @@ def response_change(self, request, obj):
             return self.add_new_version(request, obj)
 
         return super().response_change(request, obj)
+
+    def import_github_url(self, request):
```

Comment:
```suggestion
    def import_from_url_view(self, request):
```
It's nitpicking, but it's much clear now that this method is actually the django view